apps/tests/tests/e2e.test.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  paginateGetResources,
+  ResourceGroupsTaggingAPIClient,
+  ResourceTagMapping,
+} from '@aws-sdk/client-resource-groups-tagging-api';
+import type { Rule, RuleCheckResult } from '@sls-mentor/core';
+import { CustomARN } from '@sls-mentor/core';
+import * as rules from '@sls-mentor/core/src/rules';
+import type { ChecksResults } from 'sls-mentor';
+import { runSlsMentor } from 'sls-mentor/src/slsMentor';
```

Comment:
this import is weird, I think '/src/slsMentor' can be removed

Reply:
My initial intention as well :)
Unfortunately, importing from `sls-mentor` implies that code located in `packages/cli/src/index.ts` is run instantly, which triggers prompt to user while performing the tests. Which breaks the tests. This tricks ensure `commander` is not run at import time.
Additionally, commander could be removed from index